pom.xml

```diff
@@ -108,11 +108,13 @@
     <module>dubbo-plugin/dubbo-rest-spring</module>
     <module>dubbo-plugin/dubbo-triple-servlet</module>
     <module>dubbo-plugin/dubbo-triple-websocket</module>
+    <module>dubbo-plugin/dubbo-plugin-loom</module>
```

Comment:
loom is currently only supported in jdk21 and above and cannot be added to parent by default.